controllers/informationCenter/SubmissionInformationCenterHandler.inc.php

```diff
@@ -27,7 +64,7 @@ function viewInformationCenter($args, $request) {
 		$user = $request->getUser();
 		// Do not display the History tab if the user is not a manager or a sub-editor
 		$userHasRole = $user->hasRole(array(ROLE_ID_MANAGER, ROLE_ID_SUB_EDITOR), $this->_submission->getContextId());
-		$templateMgr->assign('removeHistoryTab', !$userHasRole);
+		$templateMgr->assign('removeHistoryTab', !$userHasRole || !$this->_isCurrentUserAssignedEditor);
```

Comment:
This check was probably needed because of the assistant role, but now that it is removed from the parent class?

Reply:
I'm not sure what you mean here. Are you saying that `$userHasRole` is redundant and I can just rely on `!$this->_isCurrentUserAssignedEditor`?

Reply:
I think this is closely related with the question above, so lets first see that...

Reply:
OK, then yes, we would need it... Sorry :-P